packages/node-core/src/indexer/sandbox.ts

```diff
@@ -4,7 +4,9 @@
 import path from 'path';
 import {Store} from '@subql/types';
 import {levelFilter} from '@subql/utils';
+import {build} from 'esbuild';
 import {merge} from 'lodash';
+import ts, {transpileModule} from 'typescript';
```

Comment:
We shouldn't be building any TS here. The project should already be built into JS